Sources/Turf/CoreLocation.swift

```diff
@@ -93,5 +93,12 @@ extension CLLocationCoordinate2D: Equatable {
     public func distance(to coordinate: CLLocationCoordinate2D) -> CLLocationDistance {
         return RadianCoordinate2D(self).distance(to: RadianCoordinate2D(coordinate)) * metersPerRadian
     }
+    
+    /// Returns the point midway between two coordinates measured in degrees
+    public func midpoint(to coordinate: CLLocationCoordinate2D) -> CLLocationCoordinate2D{
```

Comment:
this should be moved to `Point.midpoint(to:)` in `Point.swift`.

Reply:
I'll move it there.